### ISSUE TITLE
`group` or `groups` ?

### ISSUE THREAD

**Issue by rorynolan** [r-dbi/RMariaDB#257]
The README has `con <- dbConnect(RMariaDB::MariaDB(), group = "my-db")` but in the documentation of `RMariaDB::MariaDB()`, the argument is `groups`. This should be reconciled. `group` makes more sense to me but maybe I'm missing something?

PS I really love the package and use a lot of your work in general.

**Comment by krlmlr** [comment]
Thanks. Would you like to submit a PR?

**Comment by rorynolan** [comment]
Yes I would but it depends on what you want. Do you want it to be `groups` or `group`? 
IMO it should be `group` but it's really up to you.

**Comment by krlmlr** [comment]
Thanks. Unfortunately, changing `groups` to `group` now would be a breaking change for code that uses a named argument. If we do this, we need to keep both arguments and show a warning if the old version is used. Other than that I don't have a preference.
